src/packages.builds

```diff
@@ -15,6 +15,16 @@
     </Project>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(BuildAllPackages)' == 'false' AND '$(SkipManagedPackageBuild)' != 'true'" >
```

Comment:
In order for this condition to work you need to have initialized BuildAllPackages like https://github.com/dotnet/corefx/blob/release/2.0.0/src/packages.builds#L7

Reply:
Fixed